planReader/requirements.txt

```diff
@@ -0,0 +1,8 @@
+matplotlib
```

Comment:
maybe pin the versions ? (matplotlib==something.foo.bar)

Reply:
I'll let @doikovski select the version needed

Reply:
pip freeeze gives you the version info.

Reply:
true, since we're using conda, I just don't want to mess with the setup